### DISCUSSION TITLE
How to make a web interface

### DISCUSSION THREAD

**Discussion by MaxSjoestrand** [AxisCommunications/discussions#69] [ACAP] [answered]
Hi, I'm trying to find out how to make a web interface to interact with and configure an app after it has been installed & started on a camera.  I've looked at the example projects and searched through all the documentation I could find, but without success.
Is there some documentation I've missed? I would really appreciate being pointed in the right direction. Thanks.

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @MaxSjoestrand and sorry for the late reply. Are you using ACAP version 3 or ACAP version 4, and if it's version 4, is it the Computer Vision SDK or the Native SDK you are using?

**Reply by MaxSjoestrand** [reply]
Hi @mattias-kindborg-at-work, it is ACAP version 4, the Native SDK.

**Reply by mattias-kindborg-at-work** [reply]
We've got a [example](https://github.com/AxisCommunications/acap-native-sdk-examples/tree/main/web-server) in the [Native SDK examples repository](https://github.com/AxisCommunications/acap-native-sdk-examples) showcasing how you as a developer can have your ACAP app receive HTTP requests sent to the Axis device. Using this as a template, you ought to be able to serve your own HTML/CSS/JavaScript to any web browser.

Please note that this example might improve in the upcoming months, hopefully clarify and possibly provide guidance on how you as an app developer should think about app configuration.
